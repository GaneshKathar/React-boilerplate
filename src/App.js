import React from 'react';
import './app.css';
import logo from './assests/images/download.png';

function App() {
  return (
    <>
      <div className="title">
        <img src={logo} alt="logo" />
        Welcome to React
      </div>
    </>
  );
}

export default App;
