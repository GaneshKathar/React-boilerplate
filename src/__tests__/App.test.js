import { render } from '@testing-library/react';
import React from 'react';
import App from '../App';

describe('App', () => {
  it('should match snapshot', () => {
    const { container } = render(<App />);

    expect(container).toMatchSnapshot();
  });

  it('should have title class', () => {
    const { getByText } = render(<App />);

    expect(getByText('Welcome to React')).toHaveClass('title');
  });
});
