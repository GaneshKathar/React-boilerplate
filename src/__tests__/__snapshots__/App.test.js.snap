// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<div>
  <div
    class="title"
  >
    <img
      alt="logo"
      src="[object Object]"
    />
    Welcome to React
  </div>
</div>
`;
